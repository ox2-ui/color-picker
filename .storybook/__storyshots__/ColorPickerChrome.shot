exports[`coral`] = `
<div>
  <div
    onClick={[Function]}
    style={
      Object {
        "background": "#fff",
        "borderRadius": "1px",
        "boxShadow": "0 0 0 1px rgba(0,0,0,.1)",
        "cursor": "pointer",
        "display": "inline-block",
        "height": "20px",
        "padding": "5px",
        "width": "20px",
      }
    }>
    <div
      style={
        Object {
          "backgroundColor": "#FE8057",
          "height": "100%",
          "width": "100%",
        }
      } />
  </div>
</div>
`;

exports[`default`] = `
<div>
  <div
    onClick={[Function]}
    style={
      Object {
        "background": "#fff",
        "borderRadius": "1px",
        "boxShadow": "0 0 0 1px rgba(0,0,0,.1)",
        "cursor": "pointer",
        "display": "inline-block",
        "height": "20px",
        "padding": "5px",
        "width": "20px",
      }
    }>
    <div
      style={
        Object {
          "backgroundColor": undefined,
          "height": "100%",
          "width": "100%",
        }
      } />
  </div>
</div>
`;
