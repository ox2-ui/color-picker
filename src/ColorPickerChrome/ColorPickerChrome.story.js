import React from 'react';
import { storiesOf } from '@kadira/storybook';
import ColorPickerChrome from './ColorPickerChrome';

const handler = (value) => console.log('🍇', value); // eslint-disable-line no-console

storiesOf('ColorPickerChrome', module)
  .add('default', () => (
    <ColorPickerChrome
      onChange={handler}
      value={''}
    />
  ))
  .add('coral', () => (
    <ColorPickerChrome
      disableAlpha={true}
      onChange={handler}
      value={{ hex: '#FE8057' }}
    />
  ));
