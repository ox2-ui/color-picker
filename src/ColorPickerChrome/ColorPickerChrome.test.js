/* eslint-disable no-unused-vars */
import React from 'react';
import { shallow, render, mount } from 'enzyme';
import ColorPickerChrome from './ColorPickerChrome';

const handler = () => true;

it('renders correctly', () => {
  const wrapper = mount(
    <ColorPickerChrome
      disableAlpha={true}
      onChange={handler}
      value={{ hex: '#FE8057' }}
    />,
  );

  expect(wrapper).toMatchSnapshot();
});
