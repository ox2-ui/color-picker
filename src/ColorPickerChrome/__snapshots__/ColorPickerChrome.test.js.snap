// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ColorPickerChrome
  disableAlpha={true}
  onChange={[Function]}
  value={
    Object {
      "hex": "#FE8057",
    }
  }
>
  <div>
    <Swatch
      color="#FE8057"
      onClick={[Function]}
    >
      <div
        onClick={[Function]}
        style={
          Object {
            "background": "#fff",
            "borderRadius": "1px",
            "boxShadow": "0 0 0 1px rgba(0,0,0,.1)",
            "cursor": "pointer",
            "display": "inline-block",
            "height": "20px",
            "padding": "5px",
            "width": "20px",
          }
        }
      >
        <div
          style={
            Object {
              "backgroundColor": "#FE8057",
              "height": "100%",
              "width": "100%",
            }
          }
        />
      </div>
    </Swatch>
  </div>
</ColorPickerChrome>
`;
